addons/logistics_wirecutter/script_component.hpp

```diff
@@ -88,7 +88,7 @@
 #define CUT_TIME_ENGINEER 7.5
 
 #define HAS_WIRECUTTER(unit) (\
-    "ACE_wirecutter" in (unit call EFUNC(common,uniqueItems)) \
+    count ((unit call EFUNC(common,uniqueItems)) select {_x isKindOf ["ACE_wirecutter", configFile >> "CfgWeapons"]}) > 0 \
```

Comment:
Best approach would be to add a config entry that defines whether an item is a wirecutter so inheritance from the ACE item isn't strictly required.

Reply:
Agreed though am unsure how best to do that in a performant manner without iterating through all unique items

Reply:
Pre-compile a list of all wirecutters and check if the unit has any of them.